src/components/PreviewComment/index.js

```diff
@@ -109,12 +109,12 @@ export const styles = StyleSheet.create({
   container: (totalComment) => ({
     borderLeftWidth: 1,
     marginHorizontal: SIZES.base - 2,
-    borderLeftColor: totalComment >= 1 ? COLORS.balance_gray : COLORS.white
+    borderLeftColor: totalComment >= 1 ? COLORS.balance_gray : COLORS.almostBlack
   }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.white` to `COLORS.almostBlack` for `borderLeftColor` when `totalComment < 1` might affect the visibility of the border in dark mode. Please ensure that this color change is compatible with all themes (light and dark).

```diff
-    borderLeftColor: totalComment >= 1 ? COLORS.balance_gray : COLORS.white
+    borderLeftColor: totalComment >= 1 ? COLORS.balance_gray : COLORS.almostBlack
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->